tests/profiler/test_profiler.py

```diff
@@ -311,9 +311,14 @@ def test_pytorch_profiler_trainer_ddp(tmpdir, pytorch_profiler):
     )
     trainer.fit(model)
 
-    expected = {"validation_step"}
+    expected = {"LightningModule.BoringModel.validation_step"}
     if not _KINETO_AVAILABLE:
-        expected |= {"training_step_and_backward", "training_step", "backward"}
+        expected |= {
+            "training_step_and_backward",
+            "LightningModule.BoringModel.training_step",
+            "LightningModule.BoringModel.backward",
+        }
+    print([e.name for e in pytorch_profiler.function_events])
```

Comment:
debugging statement
```suggestion
```

Reply:
Will remove once I can get this test to pass :/ (https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/results?buildId=47638&view=logs&j=3afc50db-e620-5b81-6016-870a6976ad29&t=d9f671c5-a304-5675-5394-961fd7f98b9b)